Pipfile

```diff
@@ -10,10 +10,7 @@ sphinx = ">=4.5"
 isort = "<6.0" # Unpin when isort==6.0.0 is released
 
 [packages]
-invenio-app-rdm = {version = "~=13.0.0b1.dev6", extras = ["opensearch2"]}
-
-# NOTE: Tempoarily pinned until https://github.com/zenodo/zenodo-rdm/issues/998 is fixed
```

Comment:
changes were reverted in [`invenio-app-rdm`](https://github.com/inveniosoftware/invenio-app-rdm/commit/10afcfb741a4d1470b9d825fbb9d01c4aee4544e) and [`invenio-rdm-records`](https://github.com/inveniosoftware/invenio-rdm-records/commit/be02ccd164b8156ea3157cdc747619b8d9d0349b)